src/lib/components/CarouselCard.svelte

```diff
@@ -1,6 +1,11 @@
+<script>
+    /** @type {String} */
+    export let name = '';
+</script>
+
 <!-- Carousel Card with filler content -->
 <div class="w-56 flex-initial rounded-full bg-blue p-5 text-center font-inter">
     <!-- Sample Text -->
-    <p class="m-0">Lorem ipsum dolor sit amet consectetur</p>
+    <p class="m-0">{name}</p>
```

Comment:
Let's use slots here to give maximum flexibility to the caller. Using slots will let them use bolded text, italics, links, etc.